packages/flutter_tools/lib/src/commands/symbolize.dart

```diff
@@ -190,13 +264,37 @@ class DwarfSymbolizationService {
     required IOSink output,
     required Uint8List symbols,
   }) async {
+    await decodeWithUnits(
+        input: input,
+        output: output,
+        unitSymbols: <int, Uint8List>{
+          1: symbols,
```

Comment:
why hardcode the 1 here?

Reply:
So for an application with deferred units, the root loading unit (e.g., the one that contains the starting point of the application) always has ID 1.

`rootLoadingUnitId` is defined in `lib/src/constants.dart`, but I don't explicitly export any part of that as part of `pkg:native_stack_traces`. While we could just import that file directly to get to it, I'll make an update that officially exports it for uses outside of the Dart repo instead.

Reply:
Oh, I see, I made a constant for this locally but didn't use it here. In any case, I think it should get that constant from `pkg:native_stack_traces` anyway. (I don't expect it to ever change, but you know...)

Reply:
Okay, constant used here as well.